### ISSUE TITLE
Appropriate project name

### ISSUE THREAD

**Issue by amarjeetkapoor1** [GreatDevelopers/projectionproject1#2]
This repository and project should be given more appropriate name.

**Comment by mandeeps708** [comment]
How about `Table.js`?

**Comment by amarjeetkapoor1** [comment]
Yes, thats good I am thinking of "projection.js". Lets take more suggestions. Hope Taman and Ranvir also give suggestions as this project was started by them.
@hsrai @inderpreetsingh  Any suggestions?

**Comment by mandeeps708** [comment]
That's good too. Or combining these two: `TableProjections`.

**Comment by inderpreetsingh** [comment]
Why only english names? Try some Indian word. Also, there's no compulsion that project name should tell what project does, the word meteor doesn't give an idea that it could be a web dev framework, word ionic sounds something from chemistry.

**Comment by mandeeps708** [comment]
But it (meteor) is related somewhere to framework (maybe the workflow; not what it does). :smiling_imp:

**Comment by inderpreetsingh** [comment]
> But it (meteor) is related somewhere to framework (maybe the workflow; not what it does). :smiling_imp:

I doubt it :P I always thought because it was so different from the normal framework landscape it should have come through outer space :P

**Comment by mandeeps708** [comment]
:+1:

**Comment by amarjeetkapoor1** [comment]
@inderpreetsingh Any suggestion for some Hindi name. prakshepan.js or Prājaikaśana.js (punjabi)

**Comment by singh1114** [comment]
Anyone from table.js or projections.js.

**Comment by amarjeetkapoor1** [comment]
@singh1114  A new repository has been created for this project and its being upgrade verison of this .Why new repository was created and why is this repository not being replaced by that one if that one is the updated repository.
